scripts/python/lib/argparse_gen.py

```diff
@@ -251,7 +272,7 @@ def _check_setup(args, subparser):
 
 
 def _check_config(args, subparser):
-    if not args.mgmt_switches and not args.data_switches and args.create_container == ABSENT:
+    if not args.mgmt_switches and args.data_switches == ABSENT and args.create_container == ABSENT:
```

Comment:
Indentation is displayed correctly since this is a web tool. This is hard to read from the command line due to the width.